internal/provider/ovirt/installConfig.go

```diff
@@ -0,0 +1,45 @@
+package ovirt
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/installcfg"
+	"github.com/openshift/assisted-service/models"
+)
+
+func setPlatformValues(platform *installcfg.OvirtInstallConfigPlatform, clusterPlatform *models.OvirtPlatform) {
+	if clusterPlatform != nil {
+		platform.URL = fmt.Sprintf("https://%s/ovirt-engine/api", *clusterPlatform.Fqdn)
+		platform.Username = *clusterPlatform.Username
+		platform.Password = *clusterPlatform.Password
+		platform.Insecure = *clusterPlatform.Insecure
+		platform.CaBundle = clusterPlatform.CaBundle
+		platform.ClusterID = *clusterPlatform.ClusterID
+		platform.StorageDomainID = *clusterPlatform.StorageDomainID
+		platform.NetworkName = *clusterPlatform.NetworkName
+		platform.VnicProfileID = clusterPlatform.VnicProfileID
+	}
```

Comment:
Why don't you have the nil stuff like vsphere?
I understood from @eliorerz that it was needed if someone would have change the platform back